### DISCUSSION TITLE
Is there a good place to write a nice review of Consulo?

### DISCUSSION THREAD

**Discussion by TCROC** [consulo/consulo#440] [Q&A] [answered]
We just started using Consulo and are extremely impressed with the support and performance of the IDE.  Is there somewhere we can leave a nice review for this phenomenal IDE?

**Comment by VISTALL** [comment, marked as answer]
Idk. I'm living in not "english" speaking community. I'm not familiar with English speaking sites too much. 

Some time ago - i wrote posts at https://habr.com in russian  - it was my start point. But at current moment I'm not posting any new posts.

There few reasons. VS Code community grow up. JetBrains release Rider (I started when Rider not exists). 

I'm searching way, where Consulo can live. I dislike IDEA, that why I'm maintain also Java(and others plugins). I have big list of TODO, but nobody pay me for that (i don't want make closed-source version of Consulo).

**Reply by TCROC** [reply]
Awesome!  I'll checkout the site :)
